easybuild/easyconfigs/b/Bazel/Bazel-0.16.0-GCCcore-7.3.0.eb

```diff
@@ -12,6 +12,6 @@ sources = ['%(namelower)s-%(version)s-dist.zip']
 checksums = ['c730593916ef0ba62f3d113cc3a268e45f7e8039daf7b767c8641b6999bd49b1']
 
 builddependencies = [('binutils', '2.30')]
-dependencies = [('Java', '1.8.0_162', '', True)]
```

Comment:
Note that this might not merge if the currently pending TensorFlow 1.10 gets merged first, since it uses _172

Reply:
I'm aware of that, I'm hoping #6677 gets merged first, and then I'll resolve the conflict here.

Reply:
@boegel Doesn't look like #6677 will merge before this one.